lib/internal/bootstrap/node.js

```diff
@@ -156,13 +157,32 @@ const rawMethods = internalBinding('process_methods');
   process._getActiveHandles = rawMethods._getActiveHandles;
 
   process.getActiveResourcesInfo = function() {
+    var timeoutCount = 0;
```

Comment:
```suggestion
    let timeoutCount = 0;
```

Reply:
Ah, `var` is only needed in for loops till Node 10. Updated!